### PR TITLE
[Robot] Validate admin account, hh account name format and error notification recipient settings

### PR DESCRIPTION
**Pull request by rhakeshj**
Validate disable preferred phone enforcement shows and hides as expected

Robot WI: W-8119542

New/Existing Test - This is a new test

Verify Administrative Account Name Format and Household Account Name Format field functionality can retain values on save

Robot WI: W-8134943

New/Existing Test - This is a new test

Verify Error Notification Recipients field pick list availability

To run the test case, run this from your CCI:
cci task run robot -o suites robot/EDA/tests/browser/settings/system.robot -o include W-8119542 W-8134943

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: robot/EDA/tests/browser/settings/system.robot [hunk 1]
```diff
@@ -33,6 +54,48 @@ Verify system settings can retain value on save
...
+
+Verify Error Notification Recipients field picklist availability
+    [Documentation]         Updates 'Error Notification Recipients' to chatter group and user and
+    ...                     validates if the SFID values of the same are retained upon saving.
+    [tags]                    unstable        W-8134943     rbt:high
+    Go to EDA settings tab    System
+    Click action button on EDA settings page    Edit
+    Select recipient type value
+    ...                       Select recipient type:=Chatter Group
+    Log     ${group_name}   console=true
```
Comment (Safiyab) [inline robot/EDA/tests/browser/settings/system.robot:78]: If you want to commit this to MetaCI, I'd suggest you remove the console arg. I suggested that more for debugging on ur local. But on MetaCI, we prefer not to log to console.

File: robot/EDA/tests/browser/settings/system.robot [hunk 2]
```diff
@@ -33,6 +54,48 @@ Verify system settings can retain value on save
...
+    Select recipient
+    ...                       Select Chatter Group=${group_name}
+    Click action button on EDA settings page    Save
+    Verify dropdown value
+    ...                       Error Notification Recipients=${group_id}
+    Log     ${user_id}   console=true
```
Comment (Safiyab) [inline robot/EDA/tests/browser/settings/system.robot:84]: Same here

File: robot/EDA/resources/SystemSettingsPageObject.py [hunk 3]
```diff
@@ -32,6 +32,43 @@ def update_system_dropdown_value(self,**kwargs):
...
+        """
+        for field,value in kwargs.items():
+            locator = eda_lex_locators["eda_settings_system"]["recipient_type_value"].format(field,value)
+            self.selenium.wait_until_page_contains_element(locator,
+                                                error=f"'{value}' as dropdown value in '{field}' field is not available ")
+            self.selenium.click_element(locator)
+
+    def select_recipient(self,**kwargs):
+        """ This method will enter the account name format after selecting other in the drop down
+            Pass the expected value to be set in the input field as arguments
```
Comment (Safiyab) [inline robot/EDA/resources/SystemSettingsPageObject.py:58]: Is this documentation correct ?

**Review by Safiyab — APPROVED** [review]

File: robot/EDA/tests/browser/settings/system.robot (hunk 1 above)
Comment (rhakeshj) [reply robot/EDA/tests/browser/settings/system.robot]: Forgot to remove the debug statements and its not needed. I have removed it

File: robot/EDA/resources/SystemSettingsPageObject.py (hunk 3 above)
Comment (rhakeshj) [reply robot/EDA/resources/SystemSettingsPageObject.py]: Updated documentation to reflect the functionality

File: robot/EDA/tests/browser/settings/system.robot [hunk 4]
```diff
@@ -4,15 +4,31 @@ Resource        robot/EDA/resources/EDA.robot
...
+    [Documentation]                     Creates collaboration group and gets the SFID of group and
+    ...                                 also reads SFID of security user
+    ${group_name} =                     Set variable            Test Chatter Group
+    ${user_name} =                      Set variable            Security User
+    ${group_id}=                        API Create Collaboration Group  ${group_name}
+    ${user_id} =                        API Get ID          User     Name       Security User
+    Set Suite Variable  ${group_name}
+    Set Suite Variable  ${user_name}
+    Set Suite Variable  ${group_id}
+    Set Suite Variable  ${user_id}
```
Comment (smparedes) [inline robot/EDA/tests/browser/settings/system.robot:24]: @rhakeshj Very minor suggestion maybe align all the variables so the indent is the same. For example going from one set to the next you can do:
```
    ${user_id} =                        API Get ID          User     Name       Security User
    Set Suite Variable                  ${group_name}
```
This just looks a little cleaner.

**Review by smparedes — APPROVED** [review]
@rhakeshj Approved but made a very minor suggestion.

**Review by spinkelman — APPROVED** [review]
After Functional Review with Rakesh, this item is approved. Thanks!

**Pull request merged**